docs/deployment-process/execution-containers-for-workers/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Execution container for workers
+description: Setting an execution container for workers in steps to define your project's deployment process.
+position: 15
+---
+
+For a [step](/docs/deployment-process/steps/index.md) running on a [worker](docs/infrastructure/workers/index.md) or on [Octopus server](docs/infrastructure/workers/built-in-worker.md) that you define in your [deployment processes](/docs/deployment-process/index.md), you can select a container for the step's execution. 
+
+## Setup a container 
+
+- Set a [feed](/docs/packaging-applications/package-repositories/docker-registries/index.md) in Octopus Deploy for a Docker registry.  or link to https://octopus.com/blog/build-a-real-world-docker-cicd-pipeline#add-docker-hub-as-an-external-feed
+- Add a project and define deployment process. 
+- Select a worker pool. 
+- Select to run on a container for worker. 
+- Choose the previously added container registry. 
+- Enter an image name if already known. (e.g. octopusdeploy/step-execution:0.1.1-ubuntu.18.04)
```

Comment:
```suggestion
- Enter an image name if already known. (e.g. !docker-image <octopusdeploy/worker-tools:ubuntu.18.04>)
```

This will automatically keep the docs up to date when we add new images. [See docs](https://octopusdeploy.github.io/OctoStyle/markdown#referencing-docker-images)